recipes/pulp/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "1.6.8" %}
+
+package:
+  name: pulp
+  version: '{{ version }}'
+
+source:
+  sha256: cfecd6a78c2b08539dc29b7e8281f43d729b1089bdb5e491c3c0956aadbb3fc7
+  fn: 'PuLP-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/P/PuLP/PuLP-{{ version }}.tar.gz
+
+build:
+  entry_points:
+    - pulptest=pulp:pulpTestAll
+    - pulpdoctest=pulp:pulpDoctest
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pyparsing >=2.0.1
```

Comment:
Don't think this is actually needed at build time.

Reply:
Yes, right. Good catch!